plugins/services/src/js/components/modals/ServiceRootGroupModal/index.tsx

```diff
@@ -113,7 +150,9 @@ class ServiceRootGroupModal extends React.Component<
       isPending: false,
       expandAdvancedSettings: false,
       data: emptyGroupFormData(),
-      errors: {}
+      originalData: null,
+      errors: {},
+      isEdit: false
```

Comment:
we can set `isEdit` from props here, just remove the `_` from `_props` argument and `!!props.id`